### PR TITLE
Change Travis to Circle CI, Remove demos html, move config to origami.json

### PR DESCRIPTION
**Pull request by j-mes**
- Check for correct support link
- Change from Travis to Circle CI
- Move variables from /src/demos/config.json into origami.json
- Remove main.html from /src/demos/
- Remove /src/demos/config.json
- Remove travis.yml
- Add circle.yml
- Passed OBT verify check
- Passed OBT runServer check

This is the first housekeeping pull request. I would appreciate some feedback on this because it seems that something is triggering the failures from Travis when it's meant to be Circle CI?

** I'm going to investigate re: Circle CI documentation for this **

### CODE REVIEW

**Comment by AlbertoElias** [comment]
You'll need to enable this module in circle and disable it in travis. But it looks good!

File: origami.json [hunk 1]
```diff
@@ -10,10 +10,14 @@
         ]
     },
     "ci": {
-        "travis": "https://api.travis-ci.org/repos/Financial-Times/o-ft-icons/builds.json"
+        "circle": "https://api.travis-ci.org/repos/Financial-Times/o-ft-icons/builds.json"
     },
     "demos": [
         {
+            "name": "main",
+    			  "template": "demos/src/demo.mustache",
```
Comment (AlbertoElias) [inline origami.json]: indentation issue?

Comment (j-mes) [inline origami.json]: What's the preferred method? Spaces or tabs?

**Comment by j-mes** [comment]
The housekeeping on o-ft-icons has passed all checks and can be merged at the earliest convenience.

**Comment by AlbertoElias** [comment]
Feel free to merge and release as a patch version

**Pull request merged**